# request

---
# response
int32[] pruned 